protostar/starknet/forkable_starknet.py

```diff
@@ -62,13 +60,13 @@ async def deploy(
             disable_hint_validation=disable_hint_validation,
         )
 
-        self.cheatable_state.cheatable_carried_state.update_event_selector_to_name_map(
+        self.cheatable_state.cheatable_state.update_event_selector_to_name_map(
```

Comment:
```suggestion
        self.cheatable_starknet_state.cheatable_state.update_event_selector_to_name_map(
```

Using the same name for 2 concepts is a bit confusing.

Reply:
The tricky part is that without `cheatable` this would be `self.state.state` still.

Reply:
`Cheatable` prefix can be preserved.